doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -59,7 +59,7 @@ For details, see :ref:`nrfxlib:softdevice_controller_changelog`.
 Bluetooth mesh
 --------------
 
-|no_changes_yet_note|
+* Added support for usage of :ref:`emds_readme`. For details, see `Bluetooth mesh samples`_ and `Bluetooth libraries and services`_.
```

Comment:
Ok, we leave it for now. Maybe it looks extra odd because of only one bullet point being there with similar info. We can revisit before the release and see how it looks then. Having more bullet points in the list may change it all.